test_common.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+function test_versions() {
+  export HZ_VERSION=$1
+  export PACKAGE_VERSION=$2
+  echo "Test HZ_VERSION=$HZ_VERSION PACKAGE_VERSION=$PACKAGE_VERSION"
+
+  source ./common.sh
+
+  if [[ $DEB_PACKAGE_VERSION == "$3" && $RPM_PACKAGE_VERSION == "$4" ]]; then
+    PASSED=yes
+  else
+    PASSED=no
+  fi
+  echo "Result RELEASE_VERSION=$RELEASE_VERSION, DEB=$DEB_PACKAGE_VERSION, RPM=$RPM_PACKAGE_VERSION, PASSED=$PASSED"
+}
+
+#             HZ_VERSION     PACKAGE_VERSION DEB_PACKAGE_VERSION RPM_PACKAGE_VERSION
```

Comment:
♥️

Reply:
One more thing: could we add this test to the CI workflow?

Reply:
I will look into that in a seperate PR.